src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+import java.util.Random;
+public class Lottery {
+    static ColorSupplier colorSupplier = new ColorSupplier();
+    public static Ball getRandomBall(){
+        Ball randomBall = new Ball();
+        randomBall.setColor(colorSupplier.getRandomColor());
+        int randomIndex = new Random().nextInt(101);
```

Comment:
Instead of creating a new `Random` instance every time the method is called, you should create a single `Random` instance at the class level and reuse it. This is more efficient and prevents potential issues with random number generation.